discord/stage_instance.py

```diff
@@ -110,6 +118,11 @@ def channel(self) -> Optional[StageChannel]:
         # the returned channel will always be a StageChannel or None
         return self._state.get_channel(self.channel_id)  # type: ignore
 
+    @cached_slot_property('_cs_scheduled_event')
+    def scheduled_event(self) -> Optional[ScheduledEvent]:
+        """Optional[:class:`ScheduledEvent`]: The scheduled event that belongs to the stage instance."""
+        return self.guild.get_scheduled_event(self.scheduled_event_id)  # type: ignore
```

Comment:
A comment above explaining the type ignore would be appreciated for future maintenance.

Reply:
Done :+1: